src/helpers/api-client-helpers.ts

```diff
@@ -0,0 +1,30 @@
+import { amazonMarketplaces } from '@scaleleap/amazon-marketplaces'
+import globalAxios, { AxiosError, AxiosInstance, AxiosResponse } from 'axios'
+
+import { APIConfigurationParameters } from '../types'
+import { apiErrorFactory } from './api-error-factory'
+
+export class ApiClientHelpers {
+  static assertAxiosInstance(parameters?: APIConfigurationParameters): AxiosInstance {
```

Comment:
Why are these helpers a static method, and other helpers just functions?

Reply:
@moltar : I just want to wrap them for API parameter assertions. Therefore, I used a class with static method.

Reply:
But `src/helpers/is-json-mime.ts` is also an API parameter assertion? 

Also, those static methods aren't strictly assertions IMO. Assertion functions always throw if something is not right, or pass if something is right. There is no return.